opentsdb-ingestion.md

```diff
@@ -0,0 +1,160 @@
+## OpenTSDB Ingestion
+
+There are 2 methods for ingesting OpenTSDB data into IRONdb:
+
+1. RESTful HTTP POST of OpenTSDB JSON formatted datapoint(s)
+2. Network socket listener akin to the normal OpenTSDB telnet method
+
+For the HTTP method, POST a JSON object (or an array of JSON objects) to
+the RESTful API endpoint (see the section below - *Writing OpenTSDB with HTTP*).
+Each datapoint should be encoded as follows:
+
+```
+{
+  "metric": "metric_name",
+  "timestamp": timestamp,
+  "value": value,
+  "tags": {
+    "tag_key": "tag_value",
+    "tag_key2": "tag_value2",
+    ...
+    "tag_keyn": "tag_valuen"
+  }  
+}
+```
+
+At least one tag key/value pair is required. Multiple datapoints can be sent
+as a JSON array of the datapoints, separated by commas and the entire POST
+enclosed in square brackets.
+
+For example:
+
+```
+[{
+  "metric": "my.metric.name",
+  "timestamp": 1544678300,
+  "value": 637,
+  "tags": {
+    "datacenter": "east"
+  }
+},
+{
+  "metric": "myother.metric.name",
+  "timestamp": 1544688100,
+  "value": 3475,
+  "tags": {
+    "datacenter": "west"
+  }
+}]
+```
+
+In the case of the telnet method, telnet `put` commands in the normal OpenTSDB
+format are accepted:
+
+`put<space>metric_name<space>timestamp<space>value<space>tag_key=tag_value{<space>tag_key=tag_value...}`
+
+At least one tag key/value pair must be included.  For example:
+
+`put my.metric.name<space>1480371755<space>12345.56<space>datacenter=east`
+
+If you desire higher resolution data capture, you can suffix the timestamp with a
+period, followed by the number of milliseconds in the second, or simple just
+use 13 numeric digits without the period (the last three digits will become the
+millseconds). For example:
+
+`put my.metric.name<space>1480371964.123<space>12345.56<space>datacenter=east`
+
+Or just:
+
+`put my.metric.name<space>1480371964123<space>12345.56<space>datacenter=east`
+
+These two examples mean `123 milliseconds` into the timestamp `1480371964` or
+`November 28, 2016 10:26:04 and 123ms PM UTC`
+
+** Note that, while it resembles a floating point number, this is not a float. **
+
+For data safety reasons, we recommend that you use the RESTful POST interface to
+ingest OpenTSDB data. The network socket listener provides no feedback to the
```

Comment:
If we're POSTing we are the sender and should replace 'ingest' with 'egest' or 'send' in the context of this sentence.  It's fine in the other uses, because they are in the context of if the destination accepted it or not.

Reply:
done, fixed in this document and also in the graphite-ingestion.md where it came from